coeus-webapp/src/main/webapp/WEB-INF/tags/award/awardFandARates.tag

```diff
@@ -15,7 +15,7 @@
    - 
    - You should have received a copy of the GNU Affero General Public License
    - along with this program.  If not, see <http://www.gnu.org/licenses/>.
---%
+--%>
```

Comment:
Really weird that this would only occur in one place, no?  Any way to make sure that we haven't missed any other places where this is screwed up?